src/evaluations/run_evaluation.py

```diff
@@ -64,6 +64,10 @@
     'The name of this evaluation run.')
 flags.DEFINE_integer(
     'num_runs', None, 'The number of runs per scenario.', lower_bound=1)
+flags.DEFINE_integer(
+    'workers', 0, 
```

Comment:
num_workers to match style of flag "num_runs" immediately above.

Reply:
Resolved in 2bcd89f